### PR TITLE
tag list

### PR DESCRIPTION
**Pull request by LarryHsiao**

### CODE REVIEW

File: app/src/main/java/com/larryhsiao/nyx/android/tag/TagListFragment.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        });
+        searchMenuItem.setOnMenuItemClickListener(item -> {
+            searchView.onActionViewExpanded();
+            return false;
+        });
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
+        switch (item.getItemId()) {
```
Comment (LarryHsiao) [inline app/src/main/java/com/larryhsiao/nyx/android/tag/TagListFragment.java]: Codacy Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5112315)

**Comment by LarryHsiao** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- app/src/main/java/com/larryhsiao/nyx/android/tag/TagListFragment.java  3
- app/src/main/java/com/larryhsiao/nyx/android/tag/TagListAdapter.java  1
- app/src/main/java/com/larryhsiao/nyx/tags/TagsByKeyword.java  3
- app/src/main/java/com/larryhsiao/nyx/tags/TagRemoval.java  3
         

Clones added
============
- app/src/main/java/com/larryhsiao/nyx/android/tag/TagListFragment.java  2
- app/src/main/java/com/larryhsiao/nyx/android/jot/JotListFragment.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5112315&bid=16809433)

**Comment by 0pdd** [comment]
@LarryHsiao the puzzle [#255](https://github.com/LarryHsiao/Nyx/issues/255) is still not solved.

**Comment by 0pdd** [comment]
@LarryHsiao the only puzzle [#255](https://github.com/LarryHsiao/Nyx/issues/255) is solved here.

**Pull request closed without merging**